src/es5/validations.js

```diff
@@ -42,7 +42,7 @@ var phonePlus = function phonePlus(v) {
 exports.phonePlus = phonePlus;
 
 var phoneNoLetters = function phoneNoLetters(v) {
-  return /^[-+()0-9]*$/.test(v);
+  return /^[-+()1-9]*$/.test(v);
```

Comment:
Why it's without zeros?

Reply:
apparently it is not possible for a belgian phone number to have a `0` in it.
the validations on the server does this correctly, but there were no rules for it on the client side yet.